capella_console_client/search.py

```diff
@@ -186,6 +191,17 @@ def _get_query_payload(self, kwargs) -> DefaultDict[str, Dict[str, Any]]:
 
         return query_payload
 
+    def _get_ownership_payload(self, kwargs) -> Optional[str]:
+        ownership = kwargs.pop("ownership", None)
```

Comment:
I'd move the `.pop` up to `StacSearch.__init__ so you can drop lines 196 and 197